pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -79,6 +79,8 @@ type ResolvedPipelineRunTask struct {
 	ResolvedTaskResources *resources.ResolvedTaskResources
 	// ConditionChecks ~~TaskRuns but for evaling conditions
 	ResolvedConditionChecks TaskConditionCheckState // Could also be a TaskRun or maybe just a Pod?
+
+	Matrixed bool
```

Comment:
Do we need to introduce this new field? Can it be implied from the length of `taskRunName` that its a matrix?

Reply:
removed the field, will revisit if needed when we add the list of taskRunNames